.ci/magic-modules/create-pr.sh

```diff
@@ -116,17 +116,26 @@ if [ -n "$ANSIBLE_REPO_USER" ]; then
   # If there is now a difference in the ansible_version_added files, those
   # should be pushed back up to the user's MM branch to be reviewed.
   if git diff --name-only HEAD^1 | grep "ansible_version_added.yaml"; then
+    # Setup git config.
     git config --global user.email "magic-modules@google.com"
     git config --global user.name "Modular Magician"
 
     BRANCH=$(git config --get pullrequest.branch)
     REPO=$(git config --get pullrequest.repo)
+    CURRENT_BRANCH=$(git rev-parse HEAD)
+    # Add user's branch + checkout that branch.
+    git remote add non-gcp-push-target "git@github.com:$REPO"
+    git checkout non-gcp-push-target/$BRANCH
```

Comment:
This is likely to fail sometimes.  First, you need a `git fetch` to grab the push-target's branch.  Second, if there are any changes to any open files, this will crash.  The best approach is probably to commit _before_ the checkout, then cherry-pick that commit onto the user's branch.